package.json

```diff
@@ -131,7 +132,7 @@
     "gatsby-remark-responsive-iframe": "^2.3.1",
     "gatsby-remark-smartypants": "^2.2.1",
     "gatsby-source-filesystem": "^2.2.2",
-    "gatsby-transformer-remark": "^2.7.1",
+    "gatsby-transformer-remark": "^2.8.4",
```

Comment:
@rogermparent it won't break the deployment right?

Reply:
I'm testing it right now, but I don't believe so. The only Gatsby plugins we need pinned to old versions currently are the ones specifically related to Sharp.
If it does break anything, I'll find out soon enough.